src/sentry/migrations/0553_squash_same_name_repos.py

```diff
@@ -0,0 +1,89 @@
+# Generated by Django 3.2.20 on 2023-09-15 00:00
+
+from django.db import migrations, router, transaction
+from django.db.models import Count, Q
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+def squash_duped_repos(apps, schema_editor):
+    Repository = apps.get_model("sentry", "Repository")
+    PullRequest = apps.get_model("sentry", "PullRequest")
+    ReleaseHeadCommit = apps.get_model("sentry", "ReleaseHeadCommit")
+    LatestRepoReleaseEnvironment = apps.get_model("sentry", "LatestRepoReleaseEnvironment")
+    Commit = apps.get_model("sentry", "Commit")
+
+    duplicates = Repository.objects.values("name").annotate(count=Count("name")).filter(count__gt=1)
```

Comment:
@cathteng any idea of the performance of this query? We'll be calling it many times. I'm concerned it's going to be slow. Might be safer to do this manually by iterating through all the repos and for each one find a row of the same organization_id, name, but different id.

Reply:
i ran the equivalent in redash and it returned pretty quickly. there are 12789 duplicates as i am posting this comment